### PR TITLE
fix: removed dead code and adjusted fileExt

### PR DESCRIPTION
**Pull request by prisis**

### CODE REVIEW

**Comment by Shinigami92** [comment]
Die to all the failing CI, I convert this to a draft for now

**Comment by prisis** [comment]
Sorry forgot to do it yesterday, thanks

**Comment by Shinigami92** [comment]
Should be rebased after we merged #303

File: src/system.ts [hunk 1]
```diff
@@ -160,25 +140,8 @@ export class System {
...
-    const mimeTypes = this.faker.definitions.system.mimeTypes;
-
-    Object.keys(mimeTypes).forEach((m) => {
-      const type = m.split('/')[0];
-
-      typeSet.add(type);
-
-      if (mimeTypes[m].extensions instanceof Array) {
-        mimeTypes[m].extensions.forEach((ext) => {
-          extensionSet.add(ext);
```
Comment (ST-DDT) [inline src/system.ts:174]: You missed this important line.

File: src/system.ts [hunk 2]
```diff
@@ -160,25 +140,8 @@ export class System {
...
-        });
-      }
-    });
-
-    const types = setToArray(typeSet);
     const extensions = setToArray(extensionSet);
```
Comment (ST-DDT) [inline src/system.ts:144]: Maybe use `mimeTypes.filter(t => t.extensions).flatMap(t => t.extensions)`

**Comment by ST-DDT** [comment]
You can update your PR title as there are no longer any added types.

**Review by ST-DDT — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]
Looks good to me

**Comment by Shinigami92** [comment]
@prisis You need to press the update button, cause your PRs are not editable by us

**Pull request merged**